src/NUnitEngine/nunit.engine/Internal/ResultHelper.cs

```diff
@@ -83,10 +89,10 @@ public static TestEngineResult MakePackageResult(this TestEngineResult result, s
         /// <remarks>Used by AbstractTestRunner MakePackageResult method.</remarks>
         public static TestEngineResult Merge(IList<TestEngineResult> results)
         {
-            TestEngineResult mergedResult = new TestEngineResult();
+            var mergedResult = new TestEngineResult();
 
-            foreach (TestEngineResult result in results)
-                foreach (XmlNode node in result.XmlNodes)
+            foreach (var result in results)
+                foreach (var node in result.XmlNodes)
```

Comment:
I could go either way on this, but since TestEngineResult holds both XmlNodes and string representations, I wanted to be quite clear when I wrote it.